contrib/ci/check-source.py

```diff
@@ -208,6 +208,8 @@ def _test_line_blocked_fns(self, line: str) -> None:
             "g_udev_device_get_property(": "Use fu_udev_device_read_property() instead",
             "HIDIOCSFEATURE": "Use fu_hidraw_device_set_feature() instead",
             "HIDIOCGFEATURE": "Use fu_hidraw_device_get_feature() instead",
+            "BIT_SET": "Use fu_bitwise_set() instead",
```

Comment:
How about also checking for a hard coded implementation of it?

Reply:
Good idea!. It also found a few more to fix.